docker/Dockerfile

```diff
@@ -68,11 +65,23 @@ ARG FQL_VER
 RUN go build -o /fql -ldflags="-X 'github.com/janderland/fql/internal/app.Version=${FQL_VER}'"
 
 
-# The final stage builds the 'fql' image.
+# 'fdb-installer' stage downloads the FDB client .deb for the final stage.
+FROM debian:12 AS fdb-installer
+
+RUN apt-get update && \
+    apt-get install --no-install-recommends -y curl=7.88.* && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
+
+ARG FDB_LIB_URL
+RUN curl -fsSL ${FDB_LIB_URL} -o /fdb.deb
+
+
+# The final stage builds the minimal 'fql' runtime image.
```

Comment:
Don't make a separate stage for this. Simply perform the download in the unnamed stage below.